raster/src/main/scala/geotrellis/raster/render/MultiBandJpgRenderMethods.scala

```diff
@@ -0,0 +1,54 @@
+package geotrellis.raster.render
+
+import geotrellis.raster._
+import spire.syntax.cfor._
+
+trait MultiBandJpgRenderMethods {
+
+  val tile: MultiBandTile
+
+  def renderJpg(): Jpg = {
+
+    assert(tile.bandCount == 3)
+
+    var (rMin, gMin, bMin) = (Int.MaxValue, Int.MaxValue, Int.MaxValue)
+    var (rMax, gMax, bMax) = (Int.MinValue, Int.MinValue, Int.MinValue)
+
+    tile.foreach { (band, z) =>
+      if(isData(z)) {
+        if(band == 0) {
+          if(z > rMax) { rMax = z }
+          if(z < rMin) { rMin = z }
+        } else if(band == 1) {
+          if(z > gMax) { gMax = z }
+          if(z < gMin) { gMin = z }
+        } else if(band == 2) {
+          if(z > bMax) { bMax = z }
+          if(z < bMin) { bMin = z }
+        }
+      }
+    }
+
+    val rgb =
+      tile.convert(TypeInt).combine(0, 1, 2) { (r, g, b) =>
```

Comment:
Where is this scaling logic coming from? Is that from somewhere else in the codebase? I feel like the scaling to 0 - 255 should be another operation, and renderJpg should just assume it's already pre-processed into the correct RGB values.